projects/plugins/jetpack/_inc/lib/core-api/wpcom-endpoints/class-wpcom-rest-api-v2-endpoint-admin-menu.php

```diff
@@ -99,6 +99,8 @@ public function get_item( $request ) { // phpcs:ignore Generic.CodeAnalysis.Unus
 	public function prepare_menu_for_response( array $menu ) {
 		global $submenu;
 
+		do_action( 'admin_menu_rest_api' );
```

Comment:
Since that hook name is not prefixed with `jetpack_`, I would expect this to be a Core hook. It doesn't appear to be. Do you think you could add a prefix and a docblock to this hook?